website/content/api-docs/secret/pki.mdx

```diff
@@ -1684,3 +1710,25 @@ and thus scale horizontally.
 set to false. If `generate_lease` is true the lease creation will be forwarded to
 the active node; if `no_store` is false the entire request will be forwarded to
 the active node.
+
+# Managed Key
+
+~> Note: Managed key is an Enterprise only feature.
```

Comment:
Lets use plural for the overall system, but singular in the way PKI uses them. "Managed keys are an..."

Reply:
Pretty sure I got all of these re-adjusted.